sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Data_Inventory_Report',
+        'select
+            -- Personal Info
+            users.first_name as "First Name",
+            users.last_name as "Last Name",
+            students.birth_date as "Date of Birth",
+            students.gender as "Gender",
+            sin_validations.sin as "SIN",
+            users.email as "Email address",
+            students.contact_info -> ''address'' ->> ''addressLine1'' as "Address Line 1",
+            students.contact_info -> ''address'' ->> ''addressLine2'' as "Address Line 2",
+            students.contact_info -> ''address'' ->> ''city'' as "City",
+            students.contact_info -> ''address'' ->> ''provinceState'' as "Province",
+            students.contact_info -> ''address'' ->> ''postalCode'' as "Contact Postal/Zip Code",
+            students.contact_info -> ''address'' ->> ''country'' as "Country",
+            students.contact_info ->> ''phone'' as "Contact Phone Number",
+            -- Account Info
+            (
+                select
+                    ''Y''
+                where
+                    exists (
+                        select
+                            1
+                        from
+                            sims.student_restrictions active_student_restrictions
+                        where
+                            active_student_restrictions.student_id = students.id
+                            and active_student_restrictions.is_active = true
+                    )
+            ) as "Active Restrictions Flag",
+            (
+                select
+                    ''Y''
+                where
+                    exists (
+                        select
+                            1
+                        from
+                            sims.student_restrictions inactive_student_restrictions
+                        where
+                            inactive_student_restrictions.student_id = students.id
+                            and inactive_student_restrictions.is_active <> true
+                    )
+            ) as "Previous (inactive) Restrictions Flag",
```

Comment:
for my info, what is the meaning of `Previous (inactive)`, I didn't get it. does it mean, the student had some kind of resolved restriction?

Reply:
As the spreadsheet for the columns 

| Data Element  | SIMS Database Element Name | Element Description |
|---|---|---|
| Active Restrictions Flag | Return ‘Y’ if there are any records matching the student and is_active = true	| Indicates whether there are active restrictions at the time this report was generated | 
| Previous (inactive) Restrictions Flag	 |  | Indicates whether there are previous restrictions on a student's file which are inactive as of the time this report was generated |